datasets/geoschem-input-data.yaml

```diff
@@ -0,0 +1,42 @@
+Name: GEOS-Chem Input Data
+Description: Input data for the GEOS-Chem Chemical Transport Model, includes NASA/GMAO MERRA-2 and GEOS-FP [meteorological products](https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#met), [chemistry input data](https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#chemistry-input-data), [emissions input data](https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#emis-inputs), and other smaller datasets such as model [initial conditions](https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#initial-conditions-input-data>).
+Documentation: https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/geos-chem-input-data-on-aws.html
+Contact: https://geoschem.github.io/support-team.html
+UpdateFrequency: New meteorological and emission data will be added when available.
+ManagedBy: "[Washington University in St. Louis](https://wustl.edu) and [Harvard University](https://www.harvard.edu/) and"
+Collabs:
+  ASDI:
+    Tags:
+      - air quality
+Tags:
+  - aws-pds
+  - climate
+  - weather
+  - meteorological
+  - environmental
+  - air quality
+  - chemistry
+  - atmosphere
+  - model
+License: https://geoschem.github.io/license.html
+Resources:
+  - Description: Top-level directory for all GEOS-Chem data.
+    ARN: unknown
+    Region: us-east-1
+    Type: S3 Bucket
```

Comment:
Can we add `Explore` here with the link to browse the bucket or do we not have a URL yet?

See [these lines](https://github.com/awslabs/open-data-registry/blob/cbb2a2c5d38ca4be9de3a8af1401efd38c63d8da/datasets/geoschem-nested-input-data.yaml#L27C1-L28C68) for an example.